Kudu.Core/Jobs/BaseJobRunner.cs

```diff
@@ -102,10 +89,14 @@ private void CacheJobBinaries(JobBase job, IJobLogger logger)
             {
                 try
                 {
-                    int currentHash = CalculateHashForJob(JobBinariesPath);
```

Comment:
There is a "Failed to calculate hash for WebJob" error message below that needs to be updated if we're no longer computing hashes